lighthouse-core/runner.js

```diff
@@ -345,10 +345,7 @@ class Runner {
       }
     }
 
-    return {
-      code: LHError.NO_ERROR,
-      message: '',
-    };
+    return undefined;
```

Comment:
can you add a test for this in `runner-test.js` to go with the existing "includes a top-level runtimeError when a gatherer throws one" test

Something like
```js
it('does not include a top-level runtimeError when gatherers were successful', async () => {
  const config = new Config({
    settings: {
      auditMode: __dirname + '/fixtures/artifacts/perflog/',
    },
    audits: [
      'content-width',
    ],
  });

  const {lhr} = await Runner.run(undefined, {config});
  assert.strictEqual(lhr.runtimeError, undefined);
});
```
(fails on current master and should pass with this PR?)

Reply:
Done.